test/Tests.Infrastructure/RavenTestBase.Cluster.cs

```diff
@@ -117,7 +117,7 @@ public IDisposable GetOrCreateCancellationToken(ref CancellationToken token)
             if (token.CanBeCanceled)
                 return null;
 
-            var cts = new CancellationTokenSource(TimeSpan.FromSeconds(15));
+            var cts = new CancellationTokenSource(TimeSpan.FromSeconds(30));
```

Comment:
any idea why 15 isn't but 30 is?

Reply:
Seems like in some runs the cluster observer is trying to promote a node back to orchestrator but it doesn't catch up to the leader's raft index because the leader's raft index keeps going up too

Reply:
ohhh that is interesting! 
I'm think we need to fix the observer, not to check with his current last committed index, but with the value it had in the previous report...